server/app/routes/GroupChat/GroupChat.js

```diff
@@ -0,0 +1,3 @@
+document.addEventListener("DOMContentLoaded", function() {
+var socket = io();
```

Comment:
and this file cause we dont need them anymore

Reply:
This one fixed an error with recognizing the io(), but essentially we dont need it.

Reply:
ok to the frontend then